### PR TITLE
Tweak some minor UI issues

### PR DESCRIPTION
**Pull request by a110605**
## What's change in this PR?
Tweak some stange LH UI issues listed in [[UI][IMPROVEMENT] Tweak some minor UI issues](https://github.com/longhorn/longhorn/issues/8646)

## Issue
[[UI][IMPROVEMENT] Tweak some minor UI issues](https://github.com/longhorn/longhorn/issues/8646)

## Result
See below comments

### CODE REVIEW

File: src/assets/styles/antd-customize/generic.less [hunk 1]
```diff
@@ -140,6 +140,12 @@
     color: white !important;
   }
 
+  .ant-table-empty{
```
Comment (a110605) [inline src/assets/styles/antd-customize/generic.less:143]: The strange white overlay on the fixed right column. Only happen when table is empty and has `fixed: right`.

image

After fix
Screenshot 2024-06-06 at 8 40 44 PM

File: src/components/Layout/Menu.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ const getMenus = function (menuArray, siderFold) {
             <span>
               {item.icon ? <Icon type={item.icon} /> : ''}
               {siderFold && topMenus.indexOf(item.key) >= 0 ? '' : item.name}
-              <Icon type="down" />
+              <Icon type="down" style={{ marginLeft: 5 }} />
```
Comment (a110605) [inline src/components/Layout/Menu.js:21]: After fix
Screenshot 2024-06-06 at 8 36 51 PM

File: src/routes/volume/VolumeList.js [hunk 3]
```diff
@@ -210,13 +210,21 @@ function list({
...
               {rebuildProgress}
             </div>
-            {isEncrypted ? <Tooltip title={'Encrypted Volume'}><Icon className="color-warning" style={{ marginRight: 5, marginBottom: 2 }} type="lock" /></Tooltip> : null}
+            {isEncrypted ? <Tooltip title={'Encrypted Volume'}><Icon className="color-warning" style={{ marginLeft: 5, marginRight: 5, marginBottom: 2 }} type="lock" /></Tooltip> : null}
             {statusUpgradingEngine(record)}
             { upgrade }
-            {attchedNodeIsDown ? <Tooltip title={'The attached node is down'}><Icon className="faulted" style={{ transform: 'rotate(45deg)', marginRight: 5 }} type="api" /></Tooltip> : ''}
+            {attachedNodeIsDown && (
+              <Tooltip title={'The attached node is down'}>
+                <Icon className="faulted" style={{ transform: 'rotate(45deg)', marginRight: 5, marginLeft: 5 }} type="api" />
```
Comment (a110605) [inline src/routes/volume/VolumeList.js:218]: Add some marginLeft for volume state column Icon

Screenshot 2024-06-06 at 8 42 30 PM

File: src/routes/volume/detail/index.less [hunk 4]
```diff
@@ -28,3 +28,9 @@
     display: inline-block;
   }
 }
+
+:global{
```
Comment (a110605) [inline src/routes/volume/detail/index.less:32]: Screenshot 2024-06-06 at 8 41 33 PM

**Review by torchiaf — APPROVED** [review]
LGTM

**Pull request merged**